### PR TITLE
Remove sequencer from context

### LINKED ISSUE #1060: Remove sequencer from context
our sequencer is sender, not sequencer in context and we don't need it

### PR DESCRIPTION
**Pull request by jfldde**
## Linked Issues
- Fixes #1060

### CODE REVIEW

**Review by rakanalh — APPROVED** [review]
LGTM

File: crates/sovereign-sdk/module-system/module-implementations/sov-accounts/src/hooks.rs [hunk 1]
```diff
@@ -34,10 +31,9 @@ impl<C: Context> TxHooks for Accounts<C> {
         &self,
         tx: &Transaction<C>,
         working_set: &mut WorkingSet<C>,
-        sequencer: &C::PublicKey,
+        _sequencer: &C::PublicKey,
```
Comment (eyusufatik) [inline crates/sovereign-sdk/module-system/module-implementations/sov-accounts/src/hooks.rs]: you can remove this arg

**Review by eyusufatik — DISMISSED** [review]
you guys are going to hate me for this nit

**Review by kpp — APPROVED** [review]

File: crates/sovereign-sdk/module-system/module-implementations/sov-accounts/src/hooks.rs (hunk 1 above)
Comment (jfldde) [reply crates/sovereign-sdk/module-system/module-implementations/sov-accounts/src/hooks.rs]: That's part of the trait though, let me look into it

**Pull request merged**